net/acme-common/files/run.sh

```diff
@@ -244,13 +250,13 @@ issue_cert()
     if [ "$APP" = "uacme" ]; then
 	if [ ! -f  "$STATE_DIR/private/key.pem" ]; then
 	    log "Create a new ACME account with email $ACCOUNT_EMAIL use staging=$use_staging"
-	    $ACME $debug --confdir "$STATE_DIR" $staging --yes new $ACCOUNT_EMAIL
+	    run_acme $debug --confdir "$STATE_DIR" $staging --yes new $ACCOUNT_EMAIL
```

Comment:
Why is --verbose (for uacme) in $debug, while --debug (for acme.sh) is just added to acme_args? Why not just add both to acme_args?

Reply:
I didn't test uacme with --debug, guess it won't hurt, but if running one or the other with the same args and the warning appears in logging, users could potentially think that something is wrong with the script
but still I get 

```
daemon.debug acme: Running acme as '/usr/lib/acme/acme.sh --home /etc/acme --renew -d domain.com  --debug'
daemon.err run-acme[10863]: acme: Running acme as '/usr/lib/acme/acme.sh --home /etc/acme --renew -d domain.com  --debug'
daemon.err run-acme[10863]: Unknown parameter :  --debug

```

Reply:
disabling debug prints: `run-acme[11054]: Unknown parameter :`  seems that $acme_args  needs to be added without ""

Reply:
No, I meant you could stick the --verbose --verbose into the end of $acme_args as well, instead of having a separate $debug variable just for that?